src/lib/mina_base/account_update.ml

```diff
@@ -477,8 +477,17 @@ module May_use_token = struct
 
   let deriver obj =
     let open Fields_derivers_zkapps in
-    iso_record ~of_record:As_record.to_variant ~to_record:As_record.of_variant
-      As_record.deriver obj
+    let may_use_token =
+      iso_record ~of_record:As_record.to_variant ~to_record:As_record.of_variant
+        As_record.deriver
+    in
+    needs_custom_js
```

Comment:
Could you add some information to PR description:

- What was the format of `may_use_token` field before this PR
- What will be after this PR
- What's motivation of changing the format?

Reply:
done!